### ISSUE TITLE
Training dataset availability

### ISSUE THREAD

**Issue by redna11** [nofundamental/lncRNAnet#1]
Hi, I see that you are publishing the test set for your algorithm. Would it be possible to upload the training set used to obtain the results in the paper, so that the community can repeat the experience?
That would help comparison to other models as well.
Thanks
